specification/maps/resource-manager/Microsoft.Maps/preview/2020-04-01-preview/maps-management.json

```diff
@@ -0,0 +1,1143 @@
+    },
+    "AccountNameParameter": {
+      "name": "accountName",
+      "in": "path",
+      "description": "The name of the Maps Account.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "PrivateAtlasNameParameter": {
+      "name": "privateAtlasName",
+      "in": "path",
+      "description": "The name of the Private Atlas instance.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "definitions": {
+    "Error": {
+      "type": "object",
+      "description": "This object is returned when an error occurs in the Maps API",
+      "properties": {
+        "code": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Error code."
+        },
+        "message": {
+          "type": "string",
+          "readOnly": true,
+          "description": "If available, a human readable description of the error."
+        },
+        "target": {
+          "type": "string",
+          "readOnly": true,
+          "description": "If available, the component generating the error."
+        },
+        "details": {
+          "type": "array",
+          "readOnly": true,
+          "description": "If available, a list of additional details about the error.",
+          "items": {
+            "type": "object",
+            "properties": {
+              "code": {
+                "type": "string",
+                "readOnly": true,
+                "description": "Error code."
+              },
+              "message": {
+                "type": "string",
+                "readOnly": true,
+                "description": "If available, a human readable description of the error."
+              },
+              "target": {
+                "type": "string",
+                "readOnly": true,
+                "description": "If available, the component generating the error."
+              }
+            }
+          }
+        }
+      }
+    },
+    "Resource": {
+      "description": "An Azure resource",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified resource identifier."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the Maps resource, which is unique within a Resource Group."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Azure resource type."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "PrivateAtlas": {
+      "description": "An Azure resource which represents which will provision the ability to create private location data.",
+      "type": "object",
+      "properties": {
+        "location": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The location of the resource."
+        },
```

Comment:
this is missing x-ms-mutability.  Can you inherit from the common type definitions here:  specification\common-types\resource-management\v1\types.json

Reply:
I applied inheritance everywhere, can you confirm I did it correctly? If not can you give me an example.